### PR TITLE
Feature/meeting request bug fixes

### PR DESCRIPTION
**Pull request by MaxCastelCast**
Fixes the following bugs :
- The markers and the textes under the markers are too far away in the LocationViewMap
- A toast is showed when you accept a meeting request (should be remove since it is deprecdated)
- When clicking on a marker in the MapScreen, the message is odd "accepted/cancelled meeting request" , should be replaced
- Remove the timeout cancellation for meeting request since it isn't useful for the app anymore and could cause some unexpected bugs

The new LocationViewMap : 
image

### CODE REVIEW

File: app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt [hunk 1]
```diff
@@ -128,21 +128,24 @@ class MeetingRequestService : FirebaseMessagingService() {
...
+            MeetingRequestManager.meetingRequestViewModel?.removeFromMeetingRequestInbox(
+                senderUid) {
+                  MeetingRequestManager.meetingRequestViewModel?.removeChosenLocalisation(
+                      senderUid) {
+                        MeetingRequestManager.meetingRequestViewModel?.updateInboxOfMessages {}
+                      }
+                }
           }
+          MeetingRequestManager.meetingRequestViewModel?.stopMeetingRequestTimer(senderUid, this)
+          showNotification("Cancelled meeting with $senderName", stringReason)
```
Comment (BotondAKovacs) [inline app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt:147]: Hardcoded string?

**Review by BotondAKovacs — COMMENTED** [review]
LGTM, just a question.

Can you elaborate in why we need to remove the timeout?

File: app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt (hunk 1 above)
Comment (MaxCastelCast) [reply app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt:147]: Yes, the problem is that we have the variable senderName in the string, I could add a constant for "Cancelled meeting with" and then concatenate the two strings, but i felt that this way it is cleaner, what do you think ?

Comment (BotondAKovacs) [reply app/src/main/java/com/github/se/icebreakrr/model/message/MeetingRequestService.kt:147]: I'm ambivalent. At the same time I understand you're rational but still it's a String that the users see that fits in strings.xml. Do as you think is best.

**Comment by MaxCastelCast** [comment]
About the first question : The thing is that sometimes, when i tested with you or other people, the meeting request timeout was randomely triggered for some reason even if it worked well on my emulator. I think when adding new fitures (like the cancellation when the user closes the app) is a better and a more elegant solution for cancellations than a timeout. And the 20 minute timeout became obselete since now we have the closed app cancellation, in my opinion

**Comment by MaxCastelCast** [comment]
For your second question: of cours ! 
Before : 
image
After : 
image

**Comment by BotondAKovacs** [comment]
> For your second question: of cours ! Before : image After : image

Thank! Just one last question, have you tested if it looks better with a slight gap between the box and the pin?

**Comment by BotondAKovacs** [comment]
> About the first question : The thing is that sometimes, when i tested with you or other people, the meeting request timeout was randomely triggered for some reason even if it worked well on my emulator. I think when adding new fitures (like the cancellation when the user closes the app) is a better and a more elegant solution for cancellations than a timeout. And the 20 minute timeout became obselete since now we have the closed app cancellation, in my opinion

fair enough

**Comment by MaxCastelCast** [comment]
Yes ! It is indeed a good idea ! 
image

**Review by BotondAKovacs — APPROVED** [review]
LGTM, please just put a screenshot of the new UI in the summary.

**Pull request merged**